### PR TITLE
Add translation of appendix in Simplified Chinese.

### PR DESCRIPTION
**Pull request by sunoru**
I added a simplified Chinese version of the appendix file. Hope it helps.

### CODE REVIEW

**Comment by ehashman** [comment]
Machine translation is struggling with this a bit but the general message looks right. Would appreciate another look, cc @TomorJM @AsterNighT

**Comment by sunoru** [comment]
I also added some changes to `index.zh_CN.md`. Basically they are adding whitespaces between Chinese and English characters, and the email address is corrected.

File: appendix.zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+他把决定“将唐氏综合症胎儿带入孕期”的人称为“反常（perverse）”，并说“（增加）患有唐氏综合症的人数”是“毫无美德可言的”。[12]
+他还说：“当胎儿患有唐氏综合症时，你应该把它打掉，再试一次。”[13]
+至少有一次，RMS 把养唐氏综合症患儿比作养宠物。[14]（注：14 号链接添加于 2021-03-25 11:33 EDT。）
+
+[11]: https://web.archive.org/web/20210325014348/https://stallman.org/archives/2016-jul-oct.html#31_October_2016_(Down's_syndrome)
+[12]: https://web.archive.org/web/20210325014343/https://stallman.org/archives/2015-jul-oct.html#21_October_2015_(Mistaking_a_fetus_for_a_baby)
+[13]: https://web.archive.org/web/20210325014628/https://stallman.org/archives/2016-mar-jun.html#23_April_2016_(Fetuses_with_Downs_syndrome)
+[14]: https://web.archive.org/web/20161107050933/https://stallman.org/archives/2016-jul-oct.html#31_October_2016_(Down's_syndrome)
+
+RMS 花了好几年的时间来反对使用人们的正确代词。这只不过是一种变相的跨性别恐惧症。在初版《GNU 友好交流指南》中，他写道：“在英语中，有多种方式可以用第三人称单数代词来表达性别中立，你不必使用‘they’。”[15]
```
Comment (TomorJM) [inline appendix.zh_CN.md:42]:
```suggestion
RMS 花费了数年时间发展一场反对使用"人们"这类正确代词的运动。这只不过是一种变相的跨性别恐惧症。在初版《GNU 友好交流指南》中，他写道：“在英语中，有多种方式可以用第三人称单数代词来表达性别中立，你不必使用‘they’。”[15]
```

File: appendix.zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+这段文字后来被更新了，但仍然是恐跨的。[16]
+他的网站的主页也有这么一句话：“‘They’是复数——对于单数的情况，要使用单数性别中立代词。”[17]
```
Comment (TomorJM) [inline appendix.zh_CN.md:44]:
```suggestion
他的个人网站主页也有这么一句话：“‘They’是复数——对于单数的情况，要使用单数性别中立代词。”[17]
```

**Review by TomorJM — COMMENTED** [review]
personal opinion :)

File: appendix.zh_CN.md (hunk 1 above)
Comment (sunoru) [reply appendix.zh_CN.md:42]: 这句话有点改变原意，因为原本的句子就是 people's correct pronouns，你觉得呢？

Comment (TomorJM) [reply appendix.zh_CN.md:42]: > 这句话有点改变原意，因为原本的句子就是 people's correct pronouns，你觉得呢？

yeah, correct, I feel the original meaning

Comment (capfredf) [reply appendix.zh_CN.md:42]: 其实可以用“正确的人称代词”

Comment (sunoru) [reply appendix.zh_CN.md:42]: > 其实可以用“正确的人称代词”

嗯，OK

**Comment by sunoru** [comment]
@ehashman Can you help merge this?

File: appendix.zh_CN.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+[4]: https://web.archive.org/web/20210325013706/https://stallman.org/archives/2018-jul-oct.html#23_September_2018_(Cody_Wilson)
+
+他经常反复提到：“将青少年自愿的的性行为贴上‘强奸’的标签，这样的法律是不正当的。”[5]
+他将美国法律中“将‘强奸’定义为包括与 N 岁以下的人自愿发生性关系（其中N不等）”这一条标准与苏丹法律进行了类比，并且称“这两部法律都歪曲了‘强奸’的定义”。[6]
+
+[5]: https://web.archive.org/web/20210325013844/https://stallman.org/archives/2017-sep-dec.html#13_November_2017_(Jelani_Maraj)
+[6]: https://web.archive.org/web/20210325013942/https://stallman.org/archives/2018-may-aug.html#14_May_2018_(Death_sentence_in_Sudan)
+
+对于那些与未成年发生了性关系的女性，他戏谑道：“我希望我 14 岁的时候也能有一个美女来用那种方式‘虐待’我。”[7]
+针对儿童色情问题，他很直接地说：“制作这样的照片本应该是一种犯罪。虽然它确实是一种犯罪，但这并不能成为禁止拥有照片副本的理由。”[8]
```
Comment (Artoria2e5) [inline appendix.zh_CN.md:22]: 不要换行。Markdown换行会变成多余的空格。

Comment (Artoria2e5) [reply appendix.zh_CN.md:22]: ……我想了一想，数字链接之后给空格一点都不多余。

Comment (sunoru) [reply appendix.zh_CN.md:22]: 嗯，是的，我都是特意在数字后面换的行。

**Comment by ehashman** [comment]
Closing in favour of #2498

**Pull request closed without merging**